resources/opensearchservice-vpcendpoints.go

```diff
@@ -0,0 +1,76 @@
+package resources
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/opensearchservice"
+	"github.com/rebuy-de/aws-nuke/v2/pkg/types"
+)
+
+type OSVPCEndpoint struct {
+	svc           *opensearchservice.OpenSearchService
+	vpcEndpointId *string
+}
+
+func init() {
+	register("OSVPCEndpoint", ListOSVPCEndpoints)
+}
+
+func ListOSVPCEndpoints(sess *session.Session) ([]Resource, error) {
+	svc := opensearchservice.New(sess)
+
+	vpcEndpointIds, err := getOpenSearchVpcEndpointIds(svc)
+	if err != nil {
+		return nil, err
+	}
+
+	listResp, err := svc.DescribeVpcEndpoints(&opensearchservice.DescribeVpcEndpointsInput{
+		VpcEndpointIds: vpcEndpointIds,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	resources := make([]Resource, 0)
+
+	for _, vpcEndpoint := range listResp.VpcEndpoints {
+		resources = append(resources, &OSVPCEndpoint{
+			svc:           svc,
+			vpcEndpointId: vpcEndpoint.VpcEndpointId,
+		})
+	}
```

Comment:
Thanks for your contribution! 
If we have the IDs already, why are we describing them again? To make sure they exist?

Reply:
@der-eismann apologies.  You are correct.  That should have been removed and is now updated.